build/NuSpecs/ProjectReunionFrameworkPackage.nuspec

```diff
@@ -20,10 +20,10 @@
 
 <!--    <file target="lib\uap10.0" src="$BUILDOUTPUT$\$BUILDFLAVOR$\$BUILDARCH$\Microsoft.ProjectReunion\sdk\Microsoft.ProjectReunion.winmd"/> -->
     <file target="lib\uap10.0" src="$BUILDOUTPUT$\$BUILDFLAVOR$\$BUILDARCH$\Microsoft.ProjectReunion\Microsoft.ProjectReunion.winmd"/>
-    <file target="lib\uap10.0" src="$BUILDOUTPUT$\$BUILDFLAVOR$\$BUILDARCH$\Microsoft.ProjectReunion.FrameworkPackagePRI\Microsoft.ProjectReunion.pri"/>
+<!--    <file target="lib\uap10.0" src="$BUILDOUTPUT$\$BUILDFLAVOR$\$BUILDARCH$\Microsoft.ProjectReunion.FrameworkPackagePRI\Microsoft.ProjectReunion.pri"/> -->
 
     <file target="lib\uap10.0" src="Intellisense\Microsoft.ProjectReunion.xml"/>
-    <file target="lib\uap10.0\Design" src="$BUILDOUTPUT$\$BUILDFLAVOR$\$BUILDARCH$\Microsoft.ProjectReunion.Design\Microsoft.ProjectReunion.design.dll"/>
+    <file target="lib\uap10.0" src="$BUILDOUTPUT$\$BUILDFLAVOR$\$BUILDARCH$\Microsoft.ProjectReunion\Microsoft.ProjectReunion.dll"/>
```

Comment:
This is clearly wrong (should not contain the dll, note that xaml has a m.ui.xaml.**design** dll, we don't.  We should just have nothing here.  Will fix this in next PR.

Reply:
fixed and added the static implib and SampleFlatC header.